services/frontend-service/src/ui/components/ReleaseCard/ReleaseCard.scss

```diff
@@ -58,6 +58,7 @@
 }
 
 .release-environment {
+  white-space: nowrap;
   background-color: grey;
```

Comment:
I think that if you remove this line, the hardcoded styling on [this line](https://github.com/freiheit-com/kuberpult/pull/465/files#diff-2cc10d3f45fab644c81a083925f275d8e08948d777fd450a51955faac617cf12R43) is not necessary anymore

Reply:
good catch!